src/app/services/postman-sms/postman-sms.service.ts

```diff
@@ -28,7 +36,7 @@ class PostmanSmsService {
    * SMSes will be sent using govsg sender id.
    * Messages to any member of public MUST be sent using this method.
    */
-  sendMopSms(
+  _sendMopSms(
```

Comment:
Since we are underscoring to indicate internal use, why not use `private` as well to enforce visibility?

Reply:
For testing. We'll not be able to mock this function if it is marked as private. For other languages, we usually extends this class so that we can mock protected variables. For JS, it's not as easy. 

There are tools like [`rewire`](https://github.com/jhnns/rewire) in the ecosystem, but the last I tried, there were some problems getting it to work well with our unit tests iirc.